src/hotspot/share/cds/aotConstantPoolResolver.hpp

```diff
@@ -39,7 +39,9 @@ class constantPoolHandle;
 class InstanceKlass;
 class Klass;
 
-// ClassPrelinker is used to perform ahead-of-time linking of ConstantPool entries
+template <typename T> class GrowableArray;
```

Comment:
This doesn't seem part of the rename.

Reply:
This header was using GrowableArray without declaring it. The problem is discovered after this header is moved earlier due to alphabetical sorting.

Reply:
Okay - thanks for explaining.